react/README.md

```diff
@@ -52,6 +55,21 @@ This style guide is mostly based on the standards that are currently prevalent i
     }
     ```
 
+     ```filenames in React
+    // bad
+    header.jsx
+
+    //good
+    Header.jsx
+
+
+    ```filenames in Javascript
+    //bad
+    Header.js
+    
+    //good
+    header.js
+
```

Comment:
please remove this; this is not a good thing to recommend nor will the airbnb guide likely ever recommend it.